.bit/responses/1.1-Week 1 Step 1.md

```diff
@@ -74,7 +74,7 @@ Install <a href='https://desktop.github.com/'>Github Desktop</a> and commit from
 </details>
 
 
-## **:pencil: Task 1: Create a new branch named `test`, add a sentence introducing yourself to the end of the README.md file, and commit the change to `test`. Then, make a pull request to your main branch and merge the edits.**
+## **:pencil: Create a new branch named `test`, add a paragraph introducing yourself under the `About Me` section in the `blog.md`, and commit the change to `test`. Then, make a pull request to your main branch and merge the edits.**
```

Comment:
I think we should start encouraging students to *format* things right away. Maybe instead of using "paragraph," say something like add an introduction for yourself formatted in [markdown](https://www.markdownguide.org/basic-syntax/)

Reply:
added to another issue